### ISSUE TITLE
Missing SiteTree icons when overriding icon in Page

### ISSUE THREAD

**Issue by thats4shaw** [silverstripe/silverstripe-cms#2447]
Discovered a bit of an edge case with the introduction of the use of `$icon_class` on the likes of `ErrorPage` & `RedirectorPage` in `^4.4`.

When overriding `$icon` in Page.php like so:
```private static $icon = 'app/client/images/icons/page.svg';```

Classes using the `$icon_class` appear like so:
<img width="181" alt="Screen Shot 2019-06-12 at 9 58 25 AM" src="https://user-images.githubusercontent.com/855269/59319522-18bc3180-8d1f-11e9-810e-908c5cfd65be.png">

They appear to show fine in the "add new page" screen.

I haven't had a chance to delve too far into it to understand what the actual issue is.

**Comment by robbieaverill** [comment]
So in this case your Page has a icon image but no icon class defined?

**Comment by thats4shaw** [comment]
Yep, that's right.  I've just trashed that line on the projects I was upgrading to `4.4` today as it wasn't a necessity but there might be case where somebody wants to have a different default icon.

If you set `$icon_class` without `$icon` in `Page.php` then it all works fine.

**Comment by robbieaverill** [comment]
Cool, we want to support both to be used in conjunction. Thanks for the report.

**Comment by amolswnz** [comment]
Similar issue 
```
"silverstripe/framework": "4.4.1",
"silverstripe/cms": "4.4.1",
```
The regular sitetree icons are also missing
![Screen Shot 2019-06-28 at 11 23 18 AM](https://user-images.githubusercontent.com/20012807/60307263-f6b9e480-9997-11e9-9d46-96c29bd4d457.png)

**Comment by micmania1** [comment]
This affects CWP 2.3.0 since `BasePage` defines an `$icon` static. 

The workaround I've found is:

```
CWP\CWP\PageTypes\BasePage:
  icon: false
```

This forces it to fallback to using the CSS classes.
